capa/helpers.py

```diff
@@ -148,9 +148,9 @@ def log_unsupported_format_error():
     logger.error("-" * 80)
 
 
-def log_unsupported_cape_report_error():
+def log_unsupported_cape_report_error(error: str):
     logger.error("-" * 80)
-    logger.error(" Input file is not a valid CAPE report.")
+    logger.error(f" Input file is not a valid CAPE report: {error}")
```

Comment:
show what's wrong here

Reply:
```suggestion
    logger.error(" Input file is not a valid CAPE report: %s", error, exc_info=True)
```

please use logging format string.

also consider using `exc_info=True` to show the traceback if you want all the details displayed.